src/web/app/pages-instructor/instructor-sessions-page/instructor-sessions-page.component.ts

```diff
@@ -308,8 +324,8 @@ export class InstructorSessionsPageComponent extends InstructorSessionModalPageC
 
       // begin to populate session with template
       const templateSession: TemplateSession | undefined =
-          this.feedbackSessionsService.getTemplateSessions().find(
-              (t: TemplateSession) => t.name === this.sessionEditFormModel.templateSessionName);
+        this.feedbackSessionsService.getTemplateSessions().find(
+            (t: TemplateSession) => t.name === this.sessionEditFormModel.templateSessionName);
```

Comment:
It would also be good to leave unaffected lines like these unchanged (unless it gives some linting error)